jin-sik/0557-reverse-words-in-a-string-iii/0557-reverse-words-in-a-string-iii.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @param {string} s
+ * @return {string}
+ */
+var reverseWords = function(s) {
+    const splitArr = s.split(" ")
+
+    let answer = ""
+    splitArr.forEach((item, idx) => {
+        const word = [...item].reverse().join("")
+        splitArr.length - 1 !== idx ? answer += word + " " : answer += word
```

Comment:
```suggestion
        answer += splitArr.length - 1 !== idx ? word + " " : word
```

이런식으로 사용하는 삼항연산자의 사용법을 추천드립니다.
수고하셨습니다.